fala/apps/adviser/tests/test_search_view_function.py

```diff
@@ -39,6 +39,10 @@ def test_category_search_heading_closeness_and_matching(self):
             html=True,
         )
 
+    def test_search_parameters_box_is_visible(self):
+        response = self.client.get(self.url, self.data)
+        self.assertContains(response, '<div class="laa-fala__grey-box')
```

Comment:
I think BS4 would be better here:

```
soup = bs4.BeautifulSoup(response.content)
search_params_box = soup.find("div", class_="laa-fala__grey-box")
```

And then assert it's not None. This means that it found the HTML and it's content is not empty.

If you run into trouble with that and still feel assertContains is fine, then please add the closing `>` To the HTML response check.